docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+
+import { Image, Button } from '@stardust-ui/react'
+
+import _ from 'lodash'
+
+export interface GridItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+}
+
+class GridImageItemPicker extends React.Component<GridItemProps, any> {
+  imageStyle = {
+    width: '100%',
+  }
+
+  imageButtonStyles = {
+    minWidth: '56px',
+    height: '56px',
+    padding: '0',
+    background: '#fff',
+  }
```

Comment:
Styles can be defined as `const` outside the `GridImageItemPicker` class. Please also align names `imageStyle` => `imageStyles`